CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 ## Unreleased
 
+### Fixed
+
+- Fixed file permissions for cache in core by @moredatarequired in https://github.com/wandb/wandb/pull/8362
```

Comment:
Mysterious!

> Fix `PermissionError` in `use_artifact` for artifacts downloaded with core enabled (@moredatarequired in ...)

is this correct?

Reply:
Better now?

Reply:
> Allow all users to read cache files when core is enabled

This should probably have the word "artifact" somewhere in there, and possibly mention what in the public API is affected. "cache files" seems like an artifact implementation detail.